src/test/java/de/doubleslash/keeptime/common/DateFormatterTest.java

```diff
@@ -38,4 +43,21 @@ void zeroSecondsBetweenTest() {
       final long secondsBewtweenSwitched = DateFormatter.getSecondsBewtween(endDate, startDate);
       assertThat(secondsBewtweenSwitched, Matchers.is(0l)); // why??
    }
+
+   @Test
+   void applySystemLocaleShouldSetDatePickerLocale() {
+      // GIVEN
+      Locale.setDefault(Locale.ENGLISH);
```

Comment:
this is "dangerous" as you do not reset the locale back after the test. So the next test may fail because the locale is still english.
should make sure to get the system into previous state if modifying something which may influence other tests.